src/Message.js

```diff
@@ -34,13 +34,28 @@ type MessageProps = {
 
 const supportSpeechSynthesis = () => "SpeechSynthesisUtterance" in window;
 
-const speak = (message: string, voiceLang: string) => {
+const getVoices = () => new Promise(resolve => {
+  let voices = speechSynthesis.getVoices();
+  if (voices.length) {
+    resolve(voices);
+    return;
+  }
+  speechSynthesis.onvoiceschanged = () => {
+    voices = speechSynthesis.getVoices();
+    resolve(voices);
+  }
+});
+
+
+const speak = async (message: string, voiceLang: string) => {
   const synth = window.speechSynthesis;
-  let voices = [];
-  voices = synth.getVoices();
+  const voices = await getVoices();
   const toSpeak = new SpeechSynthesisUtterance(message);
-  toSpeak.voice = voices.find(voice => voice.lang === voiceLang);
-  synth.speak(toSpeak);
+  const desiredVoice = voices.find(voice => voice.lang === voiceLang);
+  if (desiredVoice) {
```

Comment:
Yes, we can fall back to English. However, I have observed a warning I'm getting regarding the usage of `speechSynthesis.speak()`. It seems that it isn't allowed to use this method without user activation since M71, around December 2018. 
 Check: https://www.chromestatus.com/feature/5687444770914304

Reply:
So, I suggest we should add a little button per message that when clicked triggers the `speechSynthesis.speak()` function, this will be less annoying for users who don't want to be listening to every message and will be equally accessible for those who need this feature to work. 
What is your opinion on this approach? @hotzenklotz

Reply:
I don't feel strongly about that. I guess having to trigger speech output by pressing a button next every message is a bit tedious/besides the point. What I mean is, that it might feel less like an "audio" chat and rather "just a like yet another feature". 

I would leave it as is. The browser warning should only be shown when TTS was enabled by the devs, right? I think this is a design decision that chatbot designers should deal with themselves. Do they want speech or not? If yes, perhaps the first bot message should explain to enable it in Chrome...

Reply:
I agree with you. So, I'm going to change to English as the default language as we talked.